app/models/concerns/filter/filter_scopes.rb

```diff
@@ -60,20 +60,22 @@ module Filter::FilterScopes
     end
 
     private def filter_for_age(scope)
-      return scope unless @filter.age_ranges.present? && (@filter.available_age_ranges.values & @filter.age_ranges).present?
+      # The age ranges will be strings if we had saved them in the db, so we map to symbols
+      age_ranges = @filter.age_ranges.map(&:to_sym)
```

Comment:
Let's throw a select present in here
```suggestion
      age_ranges = @filter.age_ranges.select(:present?).map(&:to_sym)
```

Reply:
✅